Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/Autonomous.java

```diff
@@ -1,76 +1,47 @@
 package org.usfirst.frc.team199.Robot2018.commands;
 
+import java.util.Map;
+
 import edu.wpi.first.wpilibj.DriverStation;
 import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.WaitCommand;
 
 /**
  * Initially run during Auto. Responsible for getting input from SmartDashboard
  * and the FMS, and then calling RunAuto with the specified script.  
  */
 public class Autonomous extends CommandGroup implements AutonomousInterface {
 
-    public Autonomous() {
-    		// TODO
-    	
-        // Add Commands here:
-        // e.g. addSequential(new Command1());
-        //      addSequential(new Command2());
-        // these will run in order.
-
-        // To run multiple commands at the same time,
-        // use addParallel()
-        // e.g. addParallel(new Command1());
-        //      addSequential(new Command2());
-        // Command1 and Command2 will run in parallel.
-
-        // A command group will require all of the subsystems that each member
-        // would require.
-        // e.g. if Command1 requires chassis, and Command2 requires arm,
-        // a CommandGroup containing them would require both the chassis and the
-        // arm.
+    public Autonomous(String startPos, Map<String, String> strategies, double delay) {
+    		String scriptName = "";
+    		
+    		scriptName += startPos.substring(0, 1);
+    		
```

Comment:
Can we just get (and save) the FMS data in AutonomousInit() in Robot and then pass it into this command? That way we can use the data for other things (e.g. a widget that displays the field) and not just "forget" the direct data after this command ends?